website/frontend/src/components/HighlightsSection/index.js

```diff
@@ -15,10 +15,8 @@ const HighlightsSection = () => {
         dispatch(loadHighlightsData());
     }, [highlightsData.length]);
 
-    const firstPost = highlightsData.length;
-    const lastPost = highlightsData.length <= 3 ? 0 : highlightsData.length - 3;
-    const highlights = highlightsData.slice(lastPost, firstPost);
-
+    const highlights = highlightsData.slice(0, 3);
+    
```

Comment:
Since the entries are arranged in the order of newest created, the array slice should always be from _index 0_ to _index 3_ for three **newest** highlights at a time.